engines/stark/services/userinterface.cpp

```diff
@@ -67,9 +69,14 @@ UserInterface::~UserInterface() {
 void UserInterface::init() {
 	_cursor = new Cursor(_gfx);
 
-	_currentScreen = _gameScreen = new GameScreen(_gfx, _cursor);
+	_mainMenuScreen = new MainMenuScreen(_gfx, _cursor);
```

Comment:
`_mainMenuScreen` is never freed (memory leak).

Reply:
Fixed.